widgets/css/admin-language-switcher-style.css

```diff
@@ -0,0 +1,13 @@
+* {
```

Comment:
You'll **NEVER** use CSS global selectors in WordPress plugins.
You can broke all user's themes and plugins doing things like this.

Prefix all plugin classes with `.monk-`
And please follow the CSS code standards.